src/main/flight/position.h

```diff
@@ -22,7 +22,15 @@
 
 #include "common/time.h"
 
+#ifdef USE_VARIO
+int16_t calculateEstimatedVario(int32_t baroAlt, const uint32_t dTime);
+#endif
+
+#ifdef USE_TELEMETRY_JETIEXBUS
 #define TASK_ALTITUDE_RATE_HZ 100
+#else
+#define TASK_ALTITUDE_RATE_HZ 120
```

Comment:
This breaks the performance we need for the position estimations in BF 4.5. All tasks that share data for position estimations need to run at the same task rate (100Hz).